src/_assets/css/_header.scss

```diff
@@ -42,6 +42,11 @@
       @include media-breakpoint-up(md) {
         padding: 0 bs-spacer(4);
       }
+
+      &.active {
+        border-bottom: 0.25rem solid $site-color-primary;
+        // font-weight: $font-weight-bold;
```

Comment:
We don't make active top-level sidenav entries bold, so I don't think that we should do so for the main nav. Also, making it bold makes the entries shift and it is visually distracting.

Reply:
That sounds good. I'll merge this PR (which has to code setting the class for the active entry). @fertolg will follow through with style changes.

Reply:
Cf. #1435